app/components/Entry.tsx

```diff
@@ -35,7 +35,7 @@ const Entry = ({ data }: EntryProps): JSX.Element => {
             isLegacy={data.is_legacy}
             isNonCanadian={data.no_cdn_conf}
           />
-          <Meanings meanings={data.meanings} />
+          <Meanings word={data.headword} meanings={data.meanings} />
```

Comment:
Rather than passing the headword down to the `<Meanings />` component (and its children), what are your thoughts on grabbing it from the parsed URL segment, `$headword`? There should be a `useParams()` function available:

```
  const params = useParams()
  const headword = params.headword
```

...or something similar.

Reply:
done! Yeah this is good